klippy/extras/force_move.py

```diff
@@ -131,12 +131,11 @@ def cmd_SET_KINEMATIC_POSITION(self, gcmd):
         x = gcmd.get_float('X', curpos[0])
         y = gcmd.get_float('Y', curpos[1])
         z = gcmd.get_float('Z', curpos[2])
-        clear = gcmd.get('CLEAR', '').upper()
-        axes = ['X', 'Y', 'Z']
-        clear_axes = [axes.index(a) for a in axes if a in clear]
+        clear = gcmd.get('CLEAR', '').lower()
+        clear_axes = "".join([a for a in "xyz" if a in clear])
         logging.info("SET_KINEMATIC_POSITION pos=%.3f,%.3f,%.3f clear=%s",
                      x, y, z, ','.join((axes[i] for i in clear_axes)))
```

Comment:
Missing `axes[i]`
Possibly there should be: 
```
i for i in clear_axes
```

Reply:
Oops!  Thanks for testing and reporting.  Hopefully should be fixed now.

-Kevin